docs/scripts/prettify/prettify.js

```
@@ -0,0 +1,207 @@
+ * flag, as well as the ignore-case ("i") flag if any of the input patterns require it.
+ *
+ * Note:
+ * - The function throws an error if any of the input RegExp objects have the global or multiline flags set.
+ * - Internal helper functions convert escape sequences (e.g., octal, hexadecimal, Unicode) and reformat character
+ *   classes to support consistent matching across the combined expression.
+ *
+ * @param {RegExp[]} a - An array of RegExp objects to be combined. Each must not have global or multiline flags.
+ * @returns {RegExp} A new RegExp that matches any pattern from the input array.
+ * @throws {Error} If any input RegExp has the global or multiline flag enabled.
+ */
+function L(a){function m(a){var f=a.charCodeAt(0);if(f!==92)return f;var b=a.charAt(1);return(f=r[b])?f:"0"<=b&&b<="7"?parseInt(a.substring(1),8):b==="u"||b==="x"?parseInt(a.substring(2),16):a.charCodeAt(1)}function e(a){if(a<32)return(a<16?"\\x0":"\\x")+a.toString(16);a=String.fromCharCode(a);if(a==="\\"||a==="-"||a==="["||a==="]")a="\\"+a;return a}function h(a){for(var f=a.substring(1,a.length-1).match(/\\u[\dA-Fa-f]{4}|\\x[\dA-Fa-f]{2}|\\[0-3][0-7]{0,2}|\\[0-7]{1,2}|\\[\S\s]|[^\\]/g),a=
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)